tests/func/test_data_cloud.py

```diff
@@ -73,29 +73,31 @@ def test(self):
             self._test_cloud(config, cl)
 
 
-class TestDataCloudBase(TestDvc):
+class DataCloudBaseTestCase(TestDvc):
     def _get_cloud_class(self):
         return None
 
-    def _should_test(self):
-        return False
+    @staticmethod
+    def should_test():
+        raise NotImplementedError
```

Comment:
Looks like some of our tests depend on this returning False by default. You could do the same here for now. The patch looks great overall!

Reply:
This *base* class has `test` method, that's why it always runs (was running before as well but would get skipped). Other tests are not failing. It's better I just fix this somehow. :)

Reply:
@skshetry Your call :) But you've definitely tapped into the part of our tests that require reimplementation the most 😄 So just manage your effort and don't spend too much time, as they will be refactored later for sure. Your PR is still great though, as it makes our old tests use new helpers, which it will continue using even when refactored later.

Reply:
Done as @efiop proposed.